CodeStarterCamp_Week2/main.swift

```diff
@@ -37,5 +38,32 @@ func printLottoResult(by matchedLottoNumbers: Set<Int>) {
     }
 }
 
-let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
-compareLotto(by: myLottoNumbers)
+//let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+//compareLotto(by: myLottoNumbers)
+
+
+// STEP 3
+var lottoResults: [String: [Int]] = [:]
+
+func generateLottoRound() {
+    for round in 1...5 {
+        let lottoNumbers = generateLottoNumbers().sorted().map{ $0 }
+        saveLottoRound(of: round, result: lottoNumbers)
+    }
+}
+
+func saveLottoRound(of round: Int, result lottoNumbers: [Int]) {
+    lottoResults["\(round)회차"] = lottoNumbers
+}
+
+func printLottoRound(of round: Int) {
+    let lottoNumbers = lottoResults["\(round)회차"]
+    if let numbers: [Int] = lottoNumbers {
+        let numbersString = numbers.map{ String($0) }.joined(separator: ", ")
```

Comment:
1. `.map`과 `{` 사이에 공백이 필요해보입니다 😀 
2. [API Design Guidelines-Naming](https://www.swift.org/documentation/api-design-guidelines/#naming)을 보면
`Name variables, parameters, and associated types according to their roles, rather than their type constraints.`
라고 말하고 있습니다. `numbersString`은 역할보단 타입에 집중한 네이밍 같아요 😀

Reply:
1. `.map`과 `{` 사이에도 공백이 필요했네요! 날카로운 지적 감사합니다 🔪
2. `formattedNumbers`라는 명칭으로 변경했습니다. `numbers`로 변경할까 했으나 `if-let`의 명칭과 중복되고, 해당 명칭은 `[Int]`를 의미하는듯 하여 `formatted`를 접두어로 붙이는쪽을 택했는데 어렵네요 😵